.storybook/preview-head.html

```diff
@@ -0,0 +1,3 @@
+<script>
+  document.documentElement.setAttribute("dir", "ltr");
```

Comment:
you can probably just do this in your main decorator
```js
addDecorator(story => <div dir="ltr"><Base locale="en-US">{story()}</Base></div>);
```
Terra might log a warning, but the attribute will still work even if its not on HTML tag

Reply:
neat! will update

Reply:
just found that it doesn't work for popup/modal rendered outside the `Base` tree. `NavHelp` looks like this after the change:
<img width="392" alt="screen shot 2017-12-04 at 9 55 07 am" src="https://user-images.githubusercontent.com/5943136/33561854-68650064-d8d9-11e7-8c51-2250eb973c54.png">
So we still need to put it in html tag